pkgs/development/interpreters/ngn-k/default.nix

```diff
@@ -31,21 +31,25 @@ useStdenv.mkDerivation {
     # make self-reference for LICENSE
     substituteAllInPlace repl.k
 
+    patchShebangs a19/a.sh a20/a.sh a21/a.sh dy/a.sh e/a.sh
+
     # don't use hardcoded /bin/sh
     for f in repl.k m.c;do
       substituteInPlace "$f" --replace "/bin/sh" "${runtimeShell}"
     done
   '';
 
   makeFlags = [ "-e" ];
-  buildFlags = [ "k" ];
+  buildFlags = [ "k" "libk.so" ];
   checkTarget = "t";
   inherit doCheck;
 
   installPhase = ''
     runHook preInstall
     install -Dm755 k "$out/bin/k"
     install -Dm755 repl.k "$out/bin/k-repl"
+    install -Dm755 libk.so "$out/lib/libk.so"
```

Comment:
Does `k` link dynamically against `libk.so` or is it statically linked? If the latter is the case we should install it to a separate `lib` output.

Reply:
`k` does not link against `libk.so`.

Reply:
Nice, then it can go to a separate output (see my patch in the other thread).